Application/Dtos/Blog/CreateCommentDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Application.Dtos.Blog
+{
+    public class CreateCommentDto
+    {
+        [Required]
+        public Guid PostId { get; set; }
+        public Guid UserId { get; set; }
```

Comment:
since you don't get `UserId` from the frontend, it technically shouldn't be a part of this DTO. Pass it as an argument next to this DTO instead. Or find a way to send it from the frontend and then mark it with `[Required]`